src/runner.js

```diff
@@ -182,10 +182,25 @@ Runner.prototype.killChild = function killChild (uuid) {
     }
 
     if (typeof child.kill === 'function') {
-        child.kill('SIGTERM');
+        var isWin = /^win/.test(process.platform);
+        var killCommand = killCommand = 'kill -9 ' + child.PhantomPID;
```

Comment:
why the double assignation `var killCommand = killCommand ...`?
Is there a global variable `killCommand`?